confirmation.html

```diff
@@ -189,11 +189,11 @@
 <table class="email-content">
     <tr>
         <td>
-            <h1>Appointment booking confirmation</h1>
+            <h1>You've successfully booked an appointment</h1>
 
-            <p>Hi Liz</p>
+            <p>Dear Liz</p>
 
-            <p>Thank you for booking a telephone guidance appointment with Pension Wise.
+            <p>thank you for booking a telephone guidance appointment with Pension Wise.
```

Comment:
There's [the way](http://english.stackexchange.com/a/195332) of the queen, the president and the pope and then there's @hvonhaaren